packages/search-ui/src/__tests__/actions/trackAutocompleteSuggestionClickThrough.test.ts

```diff
@@ -0,0 +1,41 @@
+import { AutocompleteSuggestion } from "../..";
```

Comment:
I think we need to document this in `api-core-actions.md`. Something we can do in a follow up PR.

Reply:
Thanks for the spot. Will follow up tomorrow with documentation for that new action.

Reply:
done